README.md

```diff
@@ -539,6 +539,9 @@ Here are the decorators that can be applied:
 * `@workflow.defn` - Defines a workflow class
   * Must be defined on the class given to the worker (ignored if present on a base class)
   * Can have a `name` param to customize the workflow name, otherwise it defaults to the unqualified class name
+  * Can have `dynamic=True` which means all otherwise unhandled workflows fall through to this. If present, cannot have
+    `name` argument, and run method must accept a single parameter of `Sequence[temporalio.common.RawValue]` type. The
+    payload of the raw value can be converted via `workflow.payload_converter().from_payload`.
```

Comment:
Seems like we'd want `name` argument? (To the workflow input)

Reply:
We surface this via contextual workflow info, so I intentionally leave it off here (same as .NET)